commons/src/main/java/nl/nn/adapterframework/util/DateFormatUtils.java

```diff
@@ -121,15 +121,15 @@ public static String format(Date date) {
 		return date == null ? null : format(date.toInstant());
 	}
 
-	public static String format(long date) {
+	public static String format(@Nonnull long date) {
 		return format(date, FULL_GENERIC_FORMATTER);
 	}
 
-	public static String format(Instant instant) {
+	public static String format(@Nonnull Instant instant) {
 		return format(instant, FULL_GENERIC_FORMATTER);
 	}
 
-	public static String format(long date, DateTimeFormatter formatter) {
+	public static String format(@Nonnull long date, DateTimeFormatter formatter) {
```

Comment:
`@Nonnull` is niet nodig op primitive types maar misschien wel op de `DateTimeFormatter`.